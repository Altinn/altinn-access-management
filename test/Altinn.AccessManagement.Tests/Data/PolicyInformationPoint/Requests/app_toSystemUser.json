{
  "subject": {
    "id": "urn:altinn:systemuser:uuid",
    "value": "a6355a68-86b8-4344-8a81-0248cb461468"
  },
  "party": {
    "id": "urn:altinn:partyid",
    "value": "50005545"
  },
  "resource": [
    {
      "id": "urn:altinn:org",
      "value": "ttd"
    },
    {
      "id": "urn:altinn:app",
      "value": "apps-test"
    }
  ]
}