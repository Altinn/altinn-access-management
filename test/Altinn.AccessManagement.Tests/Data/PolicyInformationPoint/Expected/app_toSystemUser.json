[
  {
    "delegationchangeid": 1337,
    "resourceregistrydelegationchangeid": 0,
    "delegationchangetype": 1,
    "resourceid": "ttd/apps-test",
    "resourcetype": "AltinnAppId",
    "offeredbypartyid": 50005545,
    "fromuuid": "00000000-0000-0000-0005-000000005545",
    "fromuuidtype": 2,
    "coveredbypartyid": null,
    "coveredbyuserid": null,
    "touuid": "a6355a68-86b8-4344-8a81-0248cb461468",
    "touuidtype": 3,
    "performedbyuserid": 20000490,
    "performedbypartyid": null,
    "performedbyuuid": null,
    "performedbyuuidtype": 0,
    "blobstoragepolicypath": "ttd/apps-test/50005545/SystemUsera6355a68-86b8-4344-8a81-0248cb461468/delegationpolicy.xml",
    "blobstorageversionid": "2024-07-18T13:37:00.1337Z",
    "created": "2024-07-18T13:37:00.1337Z"
  }
]